src/javascript/_common/localize.js

```diff
@@ -15,6 +16,12 @@ const Localize = (() => {
 
         const index = text.replace(/[\s|.]/g, '_');
 
+        const lang = urlLang().toUpperCase();
+   
+        const strings = typeof texts_json !== 'undefined' ? texts_json : {};
```

Comment:
```suggestion
        const strings = texts_json ?? {};
```
is this syntax supported now?

btw, could you please explain what you are fixing here?)

Reply:
@maryia-deriv  `texts_json` actually comes from another file but it's not explicitly imported in this file. This adds a fallback for `text_json` if it's `undefined` so that translations will work properly.